mdp/test/test_VartimeSFANode.py

```diff
@@ -69,7 +69,7 @@ def test_VartimeSFANode2():
             or ((unsfa2.sf[0, 0] < 0) and (unsfa.sf[0, 0] > 0)):
         unsfa2.sf *= -1.
 
-    assert_array_almost_equal(unsfa.sf, unsfa2.sf, decimal=10)
+    assert_array_almost_equal(abs(unsfa.sf), abs(unsfa2.sf), decimal=10)
```

Comment:
Did the VartimeSFANode fail due to a sign? Or is it just in case? The above conditional was supposed to catch that, but it may have missed the high order features. I am asking be cause we could get rid of the above conditional in this case.

Reply:
The tests sometimes failes due to sign changes, yes. OK, so I'll get rid of the conditionals.